### PR TITLE
PortfolioListView: Add column with balance of reference account

### PR DESCRIPTION
**Pull request by pfalcon**
Thus, portfolio list view will now have complete information: securities account and its MTM balance, and reference account with its cash balance.

### CODE REVIEW

**Comment by pfalcon** [comment]
https://github.com/portfolio-performance/portfolio/issues/4325#issuecomment-2525082005 describes a usecase. The patch has a non-externalized string.

File: name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/PortfolioListView.java [hunk 1]
```diff
@@ -256,6 +258,23 @@ public String getText(Object element)
         }));
         portfolioColumns.addColumn(column);
 
+        column = new Column("ref_cash_bal", "Balance of reference account", SWT.RIGHT, 100); //$NON-NLS-1$
```
Comment (Nirus2000) [inline name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/PortfolioListView.java:261]: Missing translations

**Review by Nirus2000 — CHANGES_REQUESTED** [review]

File: name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/PortfolioListView.java [hunk 2]
```diff
@@ -256,6 +258,23 @@ public String getText(Object element)
...
+        column.setLabelProvider(new ColumnLabelProvider()
+        {
+            CurrencyConverter converter = new CurrencyConverterImpl(factory, getClient().getBaseCurrency());
```
Comment (buchen) [inline name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/PortfolioListView.java:264]: The converter is not used. The balance will be shown in the account currency. Is this intended?

File: name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/PortfolioListView.java (hunk 1 above)
Comment (buchen) [reply name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/PortfolioListView.java:261]: I wonder if we should re-use the label from the account view: ColumnBalance

It works nicely in German: Depotvolumen vs. Kontostand

But maybe not so well in other languages.

**Comment by buchen** [comment]
I added the translations and merged the change.

File: name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/PortfolioListView.java (hunk 2 above)
Comment (pfalcon) [reply name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/PortfolioListView.java:264]: > The converter is not used. The balance will be shown in the account currency. Is this intended?

Yeah, it's clearly was a copy-paste issue, where label provider was initially copied from security balance column, but the getText() then was copied from AccountListView, leaving converter unused.

Thanks for taking care of the patch!

**Pull request closed without merging**